kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -100,10 +103,13 @@
         isRoot: (state) => state.pageState.topic.id === state.rootTopicId,
         pageMode: getters.pageMode,
         pageName: state => state.pageName,
-        currentChannel: state => state.currentChannel,
+        currentChannelGetter: state => state.currentChannel,
```

Comment:
oh interesting - didn't realize we had both a computed attribute and a vuex getter